src/browser.ts

```diff
@@ -231,8 +241,10 @@ export class AnalyticsBrowser {
     settings: AnalyticsBrowserSettings,
     options: InitOptions = {}
   ): Promise<[Analytics, Context]> {
+    if (settings.cdnURL) setGlobalCDNUrl(settings.cdnURL) // this is an ugly side-effect, but it's for the benefits of the plugins that get their cdn via getCDN()
```

Comment:
Nit: Can you move the comment above the line it's referring to? Keeps the column count down a bit :smile: